src/algebra/pointwise.lean

```diff
@@ -376,20 +376,28 @@ variables {α : Type*} [decidable_eq α]
 instance [has_mul α] : has_mul (finset α) :=
 ⟨λ s t, (s.product t).image (λ p : α × α, p.1 * p.2)⟩
 
+@[to_additive]
 lemma mul_def [has_mul α] {s t : finset α} :
   s * t = (s.product t).image (λ p : α × α, p.1 * p.2) := rfl
 
+@[to_additive]
 lemma mem_mul [has_mul α] {s t : finset α} {x : α} :
   x ∈ s * t ↔ ∃ y z, y ∈ s ∧ z ∈ t ∧ y * z = x :=
 by { simp only [finset.mul_def, and.assoc, mem_image, exists_prop, prod.exists, mem_product] }
 
+@[to_additive]
```

Comment:
```suggestion
@[simp, norm_cast, to_additive]
```

I hope that that's the correct order.

Reply:
Done, seeems to match a handful of other files.